theme/less/_header.less

```diff
@@ -0,0 +1,69 @@
+.header-top
+{
+  padding: 25px;
+}
+
+header label select
+{
+  width: 163px;
+  color: #3A7D11;
+  font-size: 24px;
+  border: 0;
+  opacity: 0.7;
+  appearance: none;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  background: url(/images/language-arrow.png) no-repeat right;
+}
+
+header a
+{
+  text-decoration: none;
+}
+
+.header-top a
+{
+  max-width: 190px;
+}
```

Comment:
This style basically need to be updated when text changes or icon. I'm not even sure if this is necessary, I think you can remove it?

Reply:
Done.